app/dashboard/templates/profiles/header_details.html

```diff
@@ -1,7 +1,6 @@
 {% load i18n static avatar_tags add_url_schema email_obfuscator %}
 
 <h1 class="profile-header__handle">
-  @{{ profile.name }}
```

Comment:
Was this removed by mistake?

Reply:
:see_no_evil: Thanks, yes deleted by mistake. I just restored this line.